### PR TITLE
wipe: add new package

### PR DESCRIPTION
**Pull request by lnslbrty**
Maintainer: Toni Uhlig matzeton@googlemail.com
Compile tested: x86_64, debian-stable, OpenWrt SNAPSHOT r10907+8-6cea9688af
Run tested: mvebu, linksys3200ac, OpenWrt SNAPSHOT r10907+8-6cea9688af

Description:
wipe is a little command for securely erasing files from magnetic media.

### CODE REVIEW

**Comment by neheb** [comment]
https://5155-20307838-gh.circle-artifacts.com/0/home/build/build_dir/logs/package/feeds/packages/wipe/compile.txt

File: utils/wipe/Makefile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+PKG_NAME:=wipe
+PKG_VERSION:=0.24
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/berke/$(PKG_NAME)/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=aefb4403333562f9af5e3e03ecbba4b124e98788c688662240f9b04d80bfdb2f
+
+PKG_LICENSE:=GPL-2.0
```
Comment (BKPepe) [inline utils/wipe/Makefile]: This license identifier is deprecated. Can you please look at [SPDX License list](https://SPDX.org/licenses/) and correct it, please?
You can also add PKG_LICENSE_FILES:

File: utils/wipe/Makefile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+PKG_MAINTAINER:=Toni Uhlig <matzeton@googlemail.com>
```
Comment (BKPepe) [inline utils/wipe/Makefile]: Move it above PKG_LICENSE

File: utils/wipe/Makefile [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/wipe
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=System and process monitor for Linux
+  URL:=http://lambda-diode.com/software/wipe/
+  MAINTAINER:=Toni Uhlig <matzeton@googlemail.com>
```
Comment (BKPepe) [inline utils/wipe/Makefile]: This one is duplicated. You have it already stated on line 12. Remove this one.

File: utils/wipe/Makefile [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+PKG_MAINTAINER:=Toni Uhlig <matzeton@googlemail.com>
+
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/wipe
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=System and process monitor for Linux
```
Comment (BKPepe) [inline utils/wipe/Makefile]: Isn't it wrong?

Comment (lnslbrty) [reply utils/wipe/Makefile]: C&P madness

File: utils/wipe/Makefile (hunk 2 above)
Comment (lnslbrty) [reply utils/wipe/Makefile]: done

File: utils/wipe/Makefile (hunk 3 above)
Comment (lnslbrty) [reply utils/wipe/Makefile]: done

File: utils/wipe/Makefile (hunk 2 above)
Comment (lnslbrty) [reply utils/wipe/Makefile]: done

File: utils/wipe/Makefile (hunk 1 above)
Comment (lnslbrty) [reply utils/wipe/Makefile]: done

**Comment by lnslbrty** [comment]
fixed

**Comment by diizzyy** [comment]
Wouldn't it have been better to import bcwipe instead?

**Comment by lnslbrty** [comment]
not open source afaik and `wipe` does a better job then `rm` ;)

**Comment by diizzyy** [comment]
https://repology.org/project/bcwipe/versions
Looks like the source code is available as far as I can tell?

**Comment by lnslbrty** [comment]
Did not find any.
License does not LGTM:
https://gitweb.gentoo.org/repo/gentoo.git/plain/licenses/bestcrypt

**Comment by diizzyy** [comment]
The source code?
https://www.jetico.com/file-downloads/linux/BCWipe-1.9-13.tar.gz
Anyhow, just pointed out that it's most likely a more suitable tool if you actually want to wipe storage "securely".

You also have the more commonly ported nwipe (forked from DBAN) which also uses more common methods.
https://repology.org/project/nwipe/versions
https://github.com/martijnvanbrummelen/nwipe/blob/master/man/nwipe.1#L57
https://github.com/martijnvanbrummelen/nwipe/

Couldn't find any documented ones for wipe but I probably missed something obvious ;-)

Either way, thanks for your contribution.

**Comment by lnslbrty** [comment]
`nwipe` seems better maintained then `wipe`. But it does not fit my use case. I want simply securely erase files, not whole partitions.

**Pull request merged**